dataprofiler/tests/labelers/test_character_level_cnn_model.py

```diff
@@ -430,11 +428,8 @@ def test_save(self, mock_open, *mocks):
         StringIO.close(mock_file)
 
     @mock.patch("tensorflow.keras.Model.save", return_value=None)
-    @mock.patch("tensorflow.keras.models.load_model", return_value=None)
+    @mock.patch("tensorflow.keras.models.load_model", return_value=mock.Mock())
     @mock.patch("builtins.open", side_effect=mock_open)
-    @mock.patch(
-        "dataprofiler.labelers.character_level_cnn_model.callable", return_value=True
-    )
```

Comment:
mirrored fix # 2 where the actual bug occurred

Reply:
nice